routes/web.php

```diff
@@ -20,35 +25,44 @@
     Route::post('password/change', 'Auth\ChangePasswordController@update')->name('password_update');
 });
 
-Route::get('home', 'HomeController@index')->name('home');
-Route::get('profile', 'HomeController@index')->name('profile');
-Route::post('family-actions/{user}/set-father', 'FamilyActionsController@setFather')->name('family-actions.set-father');
-Route::post('family-actions/{user}/set-mother', 'FamilyActionsController@setMother')->name('family-actions.set-mother');
-Route::post('family-actions/{user}/add-child', 'FamilyActionsController@addChild')->name('family-actions.add-child');
-Route::post('family-actions/{user}/add-wife', 'FamilyActionsController@addWife')->name('family-actions.add-wife');
-Route::post('family-actions/{user}/add-husband', 'FamilyActionsController@addHusband')->name('family-actions.add-husband');
-Route::post('family-actions/{user}/set-parent', 'FamilyActionsController@setParent')->name('family-actions.set-parent');
+Route::controller(HomeController::class)->group(function () {
```

Comment:
I believe you need to update the [RouteServiceProvider.php](https://github.com/nafiesl/silsilah/blob/5482dd4ad5ebabf3d8c5f50555ec121f858154da/app/Providers/RouteServiceProvider.php#L17) as well. You need to remove the default controller namespace definition there.

On the home route, I am getting this error:
```
Target class [App\Http\Controllers\App\Http\Controllers\HomeController] does not exist.
```

![screen_2023-07-31_023](https://github.com/nafiesl/silsilah/assets/8721551/eec30694-f23f-4deb-b7f1-8595ce3ca89b)